docs/scripts/generate-sitemap-robotstxt.ts

```diff
@@ -107,7 +107,7 @@ User-agent: *
 Allow: /
 
 # Host
-Host: ui.docs.amplify.aws
+Host: ${isProd ? 'ui.docs.amplify.aws' : 'dev.ui.docs.amplify.aws'}
```

Comment:
Rather than continue to maintain this file, let's just password protect the dev site and delete this file. According to google they don't always honor the `robots.txt` anyway if someone links to us:
https://developers.google.com/search/docs/advanced/robots/intro

Reply:
Before we implement the password, should we keep the host pointing to prod like it currently is?